.eslintrc.js

```diff
@@ -126,6 +126,7 @@ module.exports = {
       node: {
         paths: ['src'],
         extensions: ['.js', '.jsx', '.ts', '.tsx'],
+        moduleDirectory: ['node_modules', 'src/'],
```

Comment:
What's this update for

Reply:
> What's this update for

I met an error before, so I added this. Now I find the error disappeared, I will remove it